### DISCUSSION TITLE
Modularize and Customize Dashboards

### DISCUSSION THREAD

**Discussion by zoltk** [Azure/missionlz#717] [Q&A] [answered]
Hello. Its a great day.

I was wondering if there is a way to create custom dashboards in Bicep in a modular fashion. For example, if I wanted create a Dashboard with 3 panes, is there a way to create them and call those panes through modules or something similar? This is useful because if different customers want to visualize different aspects of the environment, manually updating one large Dashboard bicep template is inefficient and cumbersome. Do you have any advice or suggestions concerning this.

Thank you and I am grateful

**Comment by Chambras** [comment, marked as answer]
Hi @Fatwes 
This might be out of scope of MLZ, but have you tried this? 
https://learn.microsoft.com/en-us/azure/templates/microsoft.portal/dashboards?pivots=deployment-language-bicep
